content/tests/components/common/HideFromTests.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { cleanup, render } from "@testing-library/react";
+import React from "react";
+
+import { HideFromTests } from "devtools/components/common/HideFromTests";
+
+describe("HideFromTests", () => {
+  afterEach(async () => {
+    await cleanup();
+  });
+
+  it("should hide it's children", async () => {
```

Comment:
I think we should have the opposite test as well, since we can manipualte the globals at test time. You can see how Tif messed with `__ENV__` for tests here: https://github.com/mozilla-extensions/normandy-devtools/pull/479/files#diff-2049bef6a60a06f2ebc6de2ec30167d5R78

Reply:
Oh, I was too slow. Oh well

Reply:
I'll add another test in a new PR